robottelo/ui/container.py

```diff
@@ -0,0 +1,160 @@
+
+            [
+                {'main_tab_name': 'Preliminary', 'sub_tab_name': 'Location',
+                'name': ['Default Location']},
+                {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+                'name': 'Lifecycle Environment', 'value': self.lce.name},
+                {'main_tab_name': 'Image', 'sub_tab_name': 'Docker Hub',
+                'name': 'Docker Hub Tag', 'value': 'latest'},
+                {'main_tab_name': 'Configuration', 'name': 'Memory',
+                'value': '512m'},
+                {'main_tab_name': 'Environment', 'name': 'TTY', 'value': True},
+            ]
+        """
+        self.assign_value(locators['container.resource_name'], resource_name)
+        for parameter in parameter_list:
+            if parameter['main_tab_name'] == 'Preliminary':
+                if parameter['sub_tab_name'] == 'Organization':
+                    self._configure_orgs(parameter['name'], True)
+                elif parameter['sub_tab_name'] == 'Location':
+                    self._configure_locations(parameter['name'], True)
+        self.click(locators['container.next_section'])
+        current_tab = self._form_locator_name('Content View Tab')
+        for parameter in parameter_list:
+            if parameter['main_tab_name'] == 'Image':
+                current_tab = self._form_locator_name(
+                    parameter['sub_tab_name'] + ' Tab')
+                self.click(locators[current_tab])
+                self.assign_value(
+                    locators[self._form_locator_name(parameter['name'])],
+                    parameter['value']
+                )
+        self.click(locators[current_tab + '_next'])
+        self.assign_value(locators['container.name'], name)
+        self.assign_value(locators['container.command'], command)
+        for parameter in parameter_list:
+            if parameter['main_tab_name'] == 'Configuration':
+                self.assign_value(
+                    locators[self._form_locator_name(parameter['name'])],
+                    parameter['value']
+                )
+        self.click(locators['container.next_section'])
+        self.browser.refresh()
+        for parameter in parameter_list:
+            if parameter['main_tab_name'] == 'Environment':
+                self.assign_value(
+                    locators[self._form_locator_name(parameter['name'])],
+                    parameter['value']
+                )
+        self.click(locators['container.next_section'])
+        strategy, value = locators['container.created_container_name']
+        element = self.wait_until_element((strategy, value % name))
+        if element is None:
+            raise UINoSuchElementError(
+                'Container with name {0} was not created successfully'
+                .format(name)
+            )
+
+    def search(self, resource_name, container_name):
+        """Searches for existing container from particular compute resource"""
```

Comment:
is container search unaffected by the length of the character names? (>30 or >x in general)